src/plugins/map/MQ2Map.h

```diff
@@ -136,8 +136,8 @@ extern char MapLeftClickString[MAX_CLICK_STRINGS][MAX_STRING];
 extern bool repeatMapshow;
 extern bool repeatMaphide;
 
-extern std::vector<MapFilterOption*> s_mapFilterObjectOptions;
-extern std::vector<MapFilterOption*> s_mapFilterOptions;
+static std::vector<MapFilterOption*> s_mapFilterObjectOptions;
+static std::vector<MapFilterOption*> s_mapFilterOptions;
```

Comment:
ok, so doing this is oging to put a copy of this variable in every source file. i'm pretty sure thats not what you want to do.